sympy/utilities/tests/test_lambdify.py

```diff
@@ -829,3 +830,11 @@ def test_lambdify_dummy_arg():
     d2 = Dummy('x')
     f2 = lambdify(d2, d2 + 1)
     assert f2(2) == 3
+
+def test_issue_14283():
```

Comment:
So as written now, this  test function belongs in `sympy/sympy/printing/tests/test_pycode.py`

Reply:
Done. @bjodah